src/components/widgets/ButtonGroup.js

```diff
@@ -9,7 +9,7 @@ const ButtonGroup = ({ input, options }) => (
 
   <ToggleButtonGroup type="radio" justified={true} {...input}>
     { options.map( option =>
-      <ToggleButton key={ option.uuid } value={option.uuid} style={ buttonStyle }>{ option.name }</ToggleButton>
+      <ToggleButton key={ option.uuid } value={option.uuid} style={ buttonStyle }>{ option.name || option.display }</ToggleButton>
```

Comment:
Minor comment, but for consistency across both models this should be:

{ option.display || option.name }